File: backend/enums/journey.js
// Keep these values aligned with the PostgreSQL enums in the SQL migrations.
// Changing them is a database/API change, not just a label change in the UI.
export const JOURNEY_STATUSES = ["Planning", "Traveling", "Completed"];
export const STOP_KINDS = ["Hotel", "Restaurant", "Activity"];
export const BOOKING_STATUSES = [
  "Not booked",
  "Booked",
  "No booking needed",
  "Cancelled",
];

File: backend/enums/journey.py
"""Values must match the existing PostgreSQL enum definitions."""

from enum import Enum


class JourneyStatus(str, Enum):
    PLANNING = "Planning"
    TRAVELING = "Traveling"
    COMPLETED = "Completed"


class StopKind(str, Enum):
    HOTEL = "Hotel"
    RESTAURANT = "Restaurant"
    ACTIVITY = "Activity"


class BookingStatus(str, Enum):
    NOT_BOOKED = "Not booked"
    BOOKED = "Booked"
    NOT_REQUIRED = "No booking needed"
    CANCELLED = "Cancelled"
